### PR TITLE
Added php7.1 support

### PR DESCRIPTION
**Pull request by janhryniukpgs**
@tarnawski @partikus @dominikpgs CR please.

### CODE REVIEW

**Comment by partikus** [comment]
@janhryniukpgs good job, could you switch default php version to `7.1`? so by default `./docker.sh run` will running php7.1 version instead of 7.0

**Comment by partikus** [comment]
@janhryniukpgs don't forget to update [.travis.yml](https://github.com/PGSSoft/php-docker-skeleton/blob/master/.travis.yml)

File: docker.sh [hunk 1]
```diff
@@ -79,9 +89,13 @@ if [[ $TASK_NAME == '' ]]; then
     echo -e "Available commands:";
     echo -e "'build-images' - building docker images";
     echo -e "'run' - is running dev env and attaching tty";
+    echo -e "'run-7.1' - is running dev env and attaching tty";
```
Comment (partikus) [inline docker.sh]: `7.1` should be run as default one

**Review by partikus — CHANGES_REQUESTED** [review]
- [ ] `7.1` as a default version 
- [ ] missing update of `.travis.yml` file

**Review by dominikpgs — CHANGES_REQUESTED** [review]
@janhryniukpgs consider edit README and add 7.1 note https://github.com/PGSSoft/php-docker-skeleton/blob/master/README.md

**Comment by dominikpgs** [comment]
@janhryniukpgs 👍

File: .travis.yml [hunk 2]
```diff
@@ -4,11 +4,14 @@ services: docker
 env:
   - PROJECT_COMMAND=build
   - PROJECT_COMMAND=build-coverage
+  - PPROJET_COMMAND=build-7
```
Comment (partikus) [inline .travis.yml]: typo: `PPROJET_COMMAND` => `PROJECT_COMMAND`

**Review by partikus — CHANGES_REQUESTED** [review]
```
env:
  - PROJECT_COMMAND=build-71
  - PROJECT_COMMAND=build-71-coverage
  - PROJECT_COMMAND=build-7
  - PROJECT_COMMAND=build-7-coverage
  - PROJECT_COMMAND=build-56
  - PROJECT_COMMAND=build-56-coverage
```
and we could set `build` task as an alias for `build-71`

**Comment by jhryniuk** [comment]
@partikus @dominikpgs ping

File: docker.sh [hunk 3]
```diff
@@ -78,38 +88,78 @@ function runInBackground {
...
-        buildImages "${APP_NAME}" "${APP_VERSION}" "${USERID}" "php56"
-        runBuild "${APP_NAME}:${APP_VERSION}-php56"
-        ;;
-    'run')
+    'run-7')
         buildImages "${APP_NAME}" "${APP_VERSION}" "${USERID}" "php7"
         runInBackground "${APP_NAME}:${APP_VERSION}-php7"
         ;;
-    'run-coverage')
+    'run-coverage-7')
```
Comment (partikus) [inline docker.sh]: duplicated with [149](https://github.com/PGSSoft/php-docker-skeleton/pull/14/files#diff-51df7a7eb9518d36a5475c9cff55f3cbR149) line

Comment (partikus) [inline docker.sh]: rename `run-7-coverage`, imho `coverage` should be a suffix for builds with `xdebug`

Comment (jhryniuk) [reply docker.sh]: fixed

**Comment by partikus** [comment]
@janhryniukpgs Could you update `README.md` badges icons, please? Then I will merge it

**Review by partikus — APPROVED** [review]

**Pull request merged**